sdk/clientcore/core/src/main/java/io/clientcore/core/http/pipeline/BearerTokenAuthenticationPolicy.java

```diff
@@ -1,45 +1,34 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
-package com.azure.v2.core.http.policy;
+package io.clientcore.core.http.pipeline;
 
-import com.azure.v2.core.credentials.AccessToken;
-import com.azure.v2.core.credentials.TokenCredential;
-import com.azure.v2.core.credentials.TokenRequestContext;
-import com.azure.v2.core.implementation.AccessTokenCache;
+import io.clientcore.core.credentials.AccessToken;
+import io.clientcore.core.credentials.TokenCredential;
+import io.clientcore.core.credentials.TokenRequestContext;
+import io.clientcore.core.implementation.AccessTokenCache;
 import io.clientcore.core.http.models.HttpHeaderName;
 import io.clientcore.core.http.models.HttpHeaders;
 import io.clientcore.core.http.models.HttpRequest;
 import io.clientcore.core.http.models.Response;
-import io.clientcore.core.http.pipeline.HttpCredentialPolicy;
-import io.clientcore.core.http.pipeline.HttpPipeline;
-import io.clientcore.core.http.pipeline.HttpPipelineNextPolicy;
-import io.clientcore.core.http.pipeline.HttpPipelinePolicy;
 import io.clientcore.core.instrumentation.logging.ClientLogger;
+
 import java.io.IOException;
 import java.util.Objects;
 
 /**
  * <p>The {@code BearerTokenAuthenticationPolicy} class is an implementation of the {@link HttpPipelinePolicy} interface.
  * This policy uses a {@link TokenCredential} to authenticate the request with a bearer token.</p>
  *
- * <p>This class is useful when you need to authorize requests with a bearer token from Azure. It ensures that the
+ * <p>This class is useful when you need to authorize requests with a bearer token. It ensures that the
  * requests are sent over HTTPS to prevent the token from being leaked.</p>
  *
- * <p><strong>Code sample:</strong></p>
- *
- * <p>In this example, a {@code BearerTokenAuthenticationPolicy} is created with a {@link TokenCredential} and a scope.
- * The policy can then added to the pipeline. The request sent via the pipeline will then include the
- * Authorization header with the bearer token.</p>
- *
- * <!-- src_embed com.azure.core.http.policy.BearerTokenAuthenticationPolicy.constructor -->
- * <!-- end com.azure.core.http.policy.BearerTokenAuthenticationPolicy.constructor -->
- *
- * @see HttpPipelinePolicy
- * @see TokenCredential
- * @see HttpPipeline
- * @see HttpRequest
- * @see Response
```

Comment:
Are we losing samples because we don't have the tooling setup, or because we don't want them any longer?